announce foundation (#2059)
quarkusio/quarkusio.github.io: _posts/2024-07-31-quarkus-moving-to-commonhaus.adoc

After a period of consulting with the community inside and outside Red Hat we took the decision to submit a request for the https://quarkus.io[Quarkus project] to join the https://www.commonhaus.org/[Commonhaus Foundation].

Today, we are thrilled to announce that the Quarkus project is transitioning to the Commonhaus Foundation. Quarkus grew as a research effort inside Red Hat with the goal to revolutionise Java for the Cloud Era and since 2019 when it was unveiled as a community project, Quarkus has been nurtured and supported by Red Hat. This move to Commonhaus aligns with our https://quarkus.io/blog/quarkus-in-a-foundation/[vision of fostering a more inclusive and collaborative environment].

## Why Move to a Foundation?

Quarkus has seen tremendous growth and adoption, especially with the recent advancements in Java and Kubernetes-native development. Despite our successes, our vibrant and ever-expanding Quarkus community with over 900 contributors, there is still the perception that Quarkus is too dependent on Red Hat. We aim to change this by providing a neutral ground where other organisations and contributors can feel equally valued and involved.

## Why Commonhaus?

> Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, prominent components of Quarkus.

Reviewer: ```suggestion
Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to benefit from a self-governing model. Commonhaus suits very well the fast delivery model of Quarkus. We are excited to join the other ten prominent projects that are hosted in Commonhaus, which include Hibernate and Jackson, important components of Quarkus.
```

Best not to repeat 'prominent'.

## What's Next?

Quarkus will continue to innovate and evolve. We are dedicated to making Quarkus the best framework for Java development. This transition will enable us to welcome more contributions from a diverse range of developers and organisations. We are actively working on upcoming releases and are eager to hear your ideas and feedback.

> This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the CommonHaus foundation. 

Reviewer: ```suggestion
This move is as much for Quarkus as it is for you. If you or the organisation you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the Commonhaus foundation. 
```

> CommonHaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home. 

Reviewer: ```suggestion
Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home. 
```

> Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and opensource projects like Quarkus to have a great home. 

Reviewer 2: ```suggestion
Commonhaus is currently bootstrapping, enabling it to be shaped and moulded to help you and open source projects like Quarkus to have a great home. 
```
